test/acceptance/p2p/muxer.cpp

```diff
@@ -43,6 +43,9 @@ using std::chrono_literals::operator""ms;
 
 static const size_t kServerBufSize = 10000;  // 10 Kb
 
+// verbose==true will allow to print debug output to stdout
+static const bool verbose = false;
```

Comment:
How, who and when is going to change this constant? Should we employ TRACE_DEBUG here instead of `verbose`?

Reply:
what is TRACE_DEBUG?

Reply:
ok, getenv may help in such cases